rs/state_layout/src/state_layout.rs

```diff
@@ -486,6 +486,22 @@ impl TipHandler {
         Ok(())
     }
 
+    /// Deletes snapshots from tip if they are not in ids.
```

Comment:
Personal pet peeve:
```suggestion
    /// Deletes snapshots from tip if they are not in `ids`.
```
or
```suggestion
    /// Deletes snapshots from tip if they are not among the provided IDs.
```